packages/lesmis/src/constants/page.js

```diff
@@ -112,7 +112,7 @@ export const FQS_PAGE = {
             practices” that could be shared with other cluster schools.
           </ListItem>
           <ListItem component="li">
-            those standards that need improvement -> priorities for improvement to be included in
+            those standards that need improvement -&gt priorities for improvement to be included in
```

Comment:
I'd be happy for us to disable this ESLint rule ([react/no-unescaped-entities](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md))